test/sql/copy/s3/metadata_cache.test

```diff
@@ -18,7 +27,7 @@ statement ok
 CREATE TABLE test1 as SELECT * FROM range(10,20) tbl(i);
 
 statement ok
-SET s3_secret_access_key='minio_duckdb_user_password';SET s3_access_key_id='minio_duckdb_user';SET s3_region='eu-west-1'; SET s3_endpoint='duckdb-minio.com:9000'; SET s3_use_ssl=false;
+SET s3_endpoint='duckdb-minio.com:9000'; SET s3_use_ssl=false;
```

Comment:
Could this s3 endpoint also be moved into an env variable? then we can use the env vars to easily switch between s3 environments for all tests

Reply:
sure, domain + port together or separately?

Reply:
I would say together is probably nicer

Reply:
Now that we're looking at these variables, `SET s3_use_ssl=false;` also comes up a lot
maybe this should be an environment variable?